package.json

```diff
@@ -4,6 +4,10 @@
   "version": "0.1.0",
   "repository": "https://github.com/freedomofpress/secretshare.git",
   "description": "Desktop application providing a simple UI for the Shamir secret sharing scheme.",
+  "author": {
+    "name": "Freedom of the Press Foundation",
+    "email": "sunder@freedom.press"
```

Comment:
@msheiny I'm going to change this email address to `sysadmin@freedom.press`, cool? I'd rather that than maintain a long string of aliases.